### PR TITLE
Move up to WinUI 3 Preview 3, remove workarounds in sample, centralize SDK version

### PR DESCRIPTION
**Pull request by Scottj1s**

### CODE REVIEW

File: src/Samples/WinUIDesktopSample/WinUIDesktopSample.csproj [hunk 1]
```diff
@@ -11,7 +11,8 @@
     -->
     <XamlCodeGenerationControlFlags>DoNotGenerateOtherProviders</XamlCodeGenerationControlFlags>
     <Platforms>x86;x64</Platforms>
-    <MsAppxPackageTargets>$(MSBuildThisFileDirectory)Net5_WinUI_Fixup.Targets</MsAppxPackageTargets>
+    <TargetPlatformIdentifier>Windows</TargetPlatformIdentifier>
+    <TargetPlatformVersion>10.0.19041.0</TargetPlatformVersion>
```
Comment (j0shuams) [inline src/Samples/WinUIDesktopSample/WinUIDesktopSample.csproj:15]: Do these do the same thing as if you had `net5.0-windows10.0.19041.0` in your `TargetFramework` node ?

File: src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj [hunk 2]
```diff
@@ -23,7 +23,6 @@
     <ProjectGuid>{0212a7c5-8d3f-443c-9ebc-1f28091fdf88}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>AuthoringConsumptionTest</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.19592.0</WindowsTargetPlatformVersion>
```
Comment (j0shuams) [inline src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj:26]: Is this part of removing the workaround? Or just something not needed anymore?

**Review by j0shuams — APPROVED** [review]

File: src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj (hunk 2 above)
Comment (Scottj1s) [reply src/Authoring/AuthoringConsumptionTest/AuthoringConsumptionTest.vcxproj:26]: removing duplication of that property - vcxproj's should pick it up from the central definition in src\directory.build.props

File: src/Samples/WinUIDesktopSample/WinUIDesktopSample.csproj (hunk 1 above)
Comment (Scottj1s) [reply src/Samples/WinUIDesktopSample/WinUIDesktopSample.csproj:15]: that's even better - thx!

**Pull request merged**